docs/installing-solidity.rst

```diff
@@ -35,13 +35,14 @@ npm / Node.js
 =============
 
 Use `npm` for a convenient and portable way to install `solcjs`, a Solidity compiler. The
-`solcjs` program has fewer features than all options further down this page. Our
+`solcjs` program has fewer features than the other options further down this page. Our
```

Comment:
Maybe just `` `solcjs` (has fewer features and) does not support all the options described below.``? "fewer features than the other options" sounds a bit strange.

Reply:
I think options does not refer to e.g. commandline options. I'll reword it.